packages/real/crates/real-base/src/file.rs

```diff
@@ -1,4 +1,4 @@
-#[cfg(not(feature = "wasm"))]
+#[cfg(any(not(feature = "wasm"), test))]
```

Comment:
Does this do anything outside the package?

Reply:
It does not, the tests for this crate are not there.